metals/src/main/scala/scala/meta/internal/bsp/BspConnector.scala

```diff
@@ -33,7 +33,7 @@ class BspConnector(
 
   def resolve(): BspResolvedResult = {
     resolveExplicit().getOrElse {
-      if (buildTools.isBloop) ResolvedBloop
+      if (buildTools.loadSupported().nonEmpty) ResolvedBloop
```

Comment:
`loadSupported` in builTools does pretty much exactly what we want. It checks if it's an sbt workspace, a gradle workspace, a maven workspace, or a mill workspace. Since we can handle all of those with Bloop, we then default to Bloop for all of them if there is no explicit choice made.